healthcare/api-client/v1/fhir/fhir_resources.py

```diff
@@ -0,0 +1,654 @@
+    the httplib2.Http object.
+    """
+    credential_path = os.environ["GOOGLE_APPLICATION_CREDENTIALS"]
+    scopes = ["https://www.googleapis.com/auth/cloud-platform"]
+    credentials = ServiceAccountCredentials.from_json_keyfile_name(
+        credential_path, scopes
+    )
+    http = httplib2.Http()
+    http = credentials.authorize(http=build_http())
+    http.headers = {"Content-Type": "application/fhir+json;charset=utf-8"}
+    return discovery.build("healthcare", "v1", http=http)
+
+
+# [END healthcare_get_client_fhir]
+
+
+# [START healthcare_create_resource]
+def create_patient(project_id, cloud_region, dataset_id, fhir_store_id):
+    """Creates a new Patient resource in a FHIR store."""
+    client = get_client_fhir()
+
+    fhir_store_parent = "projects/{}/locations/{}/datasets/{}".format(
+        project_id, cloud_region, dataset_id
+    )
+
+    fhir_store_name = "{}/fhirStores/{}".format(fhir_store_parent, fhir_store_id)
+
+    resource_type = "Patient"
+
+    body = {
+        "name": [{"use": "official", "family": "Smith", "given": ["Darcy"]}],
+        "gender": "female",
+        "birthDate": "1970-01-01",
+        "active": True,
+        "resourceType": "{}".format(resource_type),
+    }
+
+    request = (
+        client.projects()
+        .locations()
+        .datasets()
+        .fhirStores()
+        .fhir()
+        .create(parent=fhir_store_name, type=resource_type, body=body)
+    )
+
+    response = request.execute()
+
+    print("Created Patient resource with ID {}".format(response["id"]))
+    print(json.dumps(response, indent=2))
+
+    return response
+
+
+# [END healthcare_create_resource]
+
+
+# [START healthcare_create_encounter]
+def create_encounter(
+    project_id, cloud_region, dataset_id, fhir_store_id, patient_id,
+):
+    """Creates a new Encounter resource in a FHIR store based on a Patient."""
+    client = get_client_fhir()
+
+    fhir_store_name = "projects/{}/locations/{}/datasets/{}/fhirStores/{}".format(
+        project_id, cloud_region, dataset_id, fhir_store_id
+    )
+
+    resource_type = "Encounter"
+
+    body = {
+        "status": "finished",
+        "class": {
+            "system": "http://hl7.org/fhir/v3/ActCode",
+            "code": "IMP",
+            "display": "inpatient encounter",
+        },
+        "reason": [
+            {
+                "text": "The patient had an abnormal heart rate. She was"
+                " concerned about this."
+            }
+        ],
+        "subject": {"reference": "Patient/{}".format(patient_id)},
+        "resourceType": "Encounter",
+    }
+
+    request = (
+        client.projects()
+        .locations()
+        .datasets()
+        .fhirStores()
+        .fhir()
+        .create(parent=fhir_store_name, type=resource_type, body=body)
+    )
+    response = request.execute()
+
+    print("Created Encounter resource with ID {}".format(response["id"]))
+    print(json.dumps(response, indent=2))
+
```

Comment:
Here and there, I'd remove these 2 empty lines.

Reply:
A result of black as we discussed in the other PR. I'll try removing it when I remember to.